flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -195,9 +196,16 @@
                     .stringType()
                     .defaultValue(getDefaultFlinkImage())
                     .withDescription(
-                            "Image to use for Flink containers. "
-                                    + "The specified image must be based upon the same Apache Flink and Scala versions as used by the application. "
-                                    + "Visit https://hub.docker.com/_/flink?tab=tags for the official docker images provided by the Flink project. The Flink project also publishes docker images here: https://hub.docker.com/r/apache/flink");
+                            Description.builder()
+                                    .text(
+                                            "Image to use for Flink containers. "
+                                                    + "The specified image must be based upon the same Apache Flink and Scala versions as used by the application. "
+                                                    + "Visit %s for the official docker images provided by the Flink project. The Flink project also publishes docker images to %s.",
+                                            link("https://hub.docker.com/_/flink?tab=tags", "here"),
+                                            link(
+                                                    "https://hub.docker.com/r/apache/flink",
```

Comment:
```suggestion
                                                    "https://hub.docker.com/r/apache/flink?tab=tags",
```

nit: Maybe also add the "?tab=tags" is better to  align with official image link.

Reply:
Ok, but I tried it and found that [https://hub.docker.com/r/apache/flink?tab=tags](https://hub.docker.com/r/apache/flink?tab=tags) is not align with the [https://hub.docker.com/_/flink?tab=tags](https://hub.docker.com/_/flink?tab=tags), maybe the correct link should be [https://hub.docker.com/r/apache/flink/tags?](https://hub.docker.com/r/apache/flink/tags?)

Reply:
Strange. But you are right. Could you update it?

Reply:
Done. Take a look, THX!